### DISCUSSION TITLE
halo应用市场的主题安装似乎有bug

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#6938] [Q&A]
有的主题我已经安装了，但是依然不显示已安装，然后我点击安装的时候却显示已经安装，是否重新安装，点击安装之后，依然不显示已安装。
![QQ20241024-160633](https://github.com/user-attachments/assets/8efea82c-7dfb-4268-9c99-04d294695e67)

**Comment by JohnNiang** [comment]
建议去主题列表中是否有灰色小点，如果有的话，代表当前这个主题是从“非应用市场”的方式安装的。

<img width="946" alt="image" src="https://github.com/user-attachments/assets/649da748-af99-4bc8-a95d-3e31000db9c7">

**Reply by IshuaWang** [reply]
我就是在应用市场安装的啊，不是导入的

**Reply by ruibaby** [reply]
建议在安装过程中打开浏览器控制台查看网络请求，看看是否有请求失败的接口。

**Comment by QingFeng-awa** [comment]
这不是 #6746 修过的问题么
